amoro-web/src/views/optimize/components/List.vue

```diff
@@ -84,6 +99,8 @@ async function getTableList() {
       tableSearchInput: tableSearchInput.value || '',
       page: pagination.current,
       pageSize: pagination.pageSize,
+      action: action.value,
```

Comment:
Is this meant to be actions?

Reply:
Yes, it is the data selected by action multiple select.